ray_on_golem/server/services/ray.py

```diff
@@ -60,6 +65,8 @@ def __init__(
         self._nodes_id_counter = 0
 
         self._create_node_tasks: Dict[NodeId, asyncio.Task] = {}
+        self._node_monitoring_tasks: Dict[NodeId, asyncio.Task] = {}
+        self._node_monitoring_timeout: timedelta = DEFAULT_NODE_MONITORING_TIMEOUT
```

Comment:
If this variable is not configured by arguments it is better to use global constant instead.

Reply:
Added argument to init